package-system/squish-ccr/build_config.json

```diff
@@ -55,24 +55,22 @@
         }
       },
       "Linux":{
-         "Linux":{
-            "custom_cmake_install":true,
-            "cmake_generate_args_release": [
-                "-G",
-                "Unix\\ Makefiles",
-                "-DCMAKE_C_COMPILER=clang-6.0",
-                "-DCMAKE_CXX_COMPILER=clang++-6.0",
-                "-DCMAKE_CXX_FLAGS=\"-fPIC -O2\"",
-                "-DCMAKE_CXX_STANDARD=17",
-                "-DCMAKE_BUILD_TYPE=Release"
-            ],
-            "cmake_build_args":[
-               "-j"
-            ],
-            "build_configs":[
-                "Release"
-            ]
-         }
-      }
+            "Linux":{
+               "custom_build_cmd": [
+                    "./build-linux-x86_64.sh"
```

Comment:
In order to reduce unnecessary scripts, i'll update the pull_and_build_from_git script to accept arguments to the 'custom_build_cmd' and adjust this accordingly